### PR TITLE
Nsng 144 visual fixes

### PR DESCRIPTION
**Pull request by igor-kamil**
Snímka obrazovky 2022-12-09 o 0 48 54

Snímka obrazovky 2022-12-09 o 0 49 11

Snímka obrazovky 2022-12-09 o 0 50 17

### CODE REVIEW

File: resources/js/components/svg/Logo.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
         />
     </svg>
 </template>
```
Comment (rastislav-chynoransky) [inline resources/js/components/svg/Logo.vue:7]: Why? I used the original logo from SNG website. This one got weird curves and stroke.

**Review by rastislav-chynoransky — CHANGES_REQUESTED** [review]
Ok, I don't have the capacity to check those changes, we should start doing it in a separate commit.

File: resources/js/components/svg/Logo.vue (hunk 1 above)
Comment (rastislav-chynoransky) [reply resources/js/components/svg/Logo.vue:7]: Ah, thank for the explanation. The close button in About will need a fix then.

Comment (igor-kamil) [reply resources/js/components/svg/Logo.vue:7]: fixed ✅ 
also SNG logo in about:
Snímka obrazovky 2022-12-09 o 11 49 29

proposed design (from figma):
Snímka obrazovky 2022-12-09 o 11 50 00

File: resources/js/components/ConfirmButton.vue [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <template>
     <button
-        class="border-black rounded-xl border-2 py-3 w-full font-bold active:bg-black active:text-white flex items-center justify-center text-lg">
+        class="flex w-full items-center justify-center rounded-xl border-2 border-black py-3 text-lg font-medium active:bg-black active:text-white"
```
Comment (rastislav-chynoransky) [inline resources/js/components/ConfirmButton.vue:3]: Should `font-medium` really apply for all contexts?

File: resources/js/views/ItemDetail.vue [hunk 3]
```diff
@@ -66,21 +42,15 @@
...
-    <div
-        class="pointer-events-none fixed bottom-0 h-24 w-full bg-gradient-to-t from-white to-transparent md:max-w-lg"
-    >
+    <div class="pointer-events-none fixed bottom-0 h-24 w-full bg-gradient-to-t from-white to-transparent md:max-w-lg">
         <div class="p-4 pt-8">
             <div class="pointer-events-auto flex space-x-4">
                 <HistoryBack v-slot="{ back }">
                     <ConfirmButton class="group bg-white" @click="back">
-                        <SvgBack class="mr-2 group-active:stroke-white" />
+                        <SvgBack class="mr-4 group-active:stroke-white" />
```
Comment (rastislav-chynoransky) [inline resources/js/views/ItemDetail.vue:53]: This should stay `mr-2` and the button should be enlarged to 32x32px.

File: resources/js/views/ItemDetail.vue [hunk 4]
```diff
@@ -90,14 +60,8 @@
...
-                <ConfirmButton
-                    v-else
-                    class="bg-green"
-                    @click="addItemFavourited(item)"
-                >
-                    <svg
-                        class="mr-3 h-[22px] w-[25px] fill-green stroke-black stroke-2"
-                    >
+                <ConfirmButton v-else class="bg-green" @click="addItemFavourited(item)">
+                    <svg class="mr-2.5 h-[22px] w-[25px] fill-green stroke-black stroke-2">
```
Comment (rastislav-chynoransky) [inline resources/js/views/ItemDetail.vue:64]:
```suggestion
                    <svg class="mr-2 h-[22px] w-[25px] fill-green stroke-black stroke-2">
```

File: resources/js/components/svg/Close.vue [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <template>
-    <svg class="h-[32px] w-[32px] stroke-2 stroke-black [stroke-linecap:round]">
-        <path d="M25 7L7 25" />
-        <path d="M25 25L7 7" />
+    <svg class="h-[20px] w-[20px] stroke-black stroke-2 [stroke-linecap:round] [stroke-linejoin:round]">
+        <path d="M19 1L1 19" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+        <path d="M19 19L1 1" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
```
Comment (rastislav-chynoransky) [inline resources/js/components/svg/Close.vue:4]: We don't need the attribute styling.

File: resources/js/components/About.vue [hunk 6]
```diff
@@ -2,11 +2,11 @@
     <Transition :duration="300">
         <div class="fixed inset-0 z-10 overflow-hidden md:mx-auto md:max-w-lg" v-if="opened">
             <div class="h-full overflow-y-auto bg-white pt-[48px] duration-300">
-                <div class="border-b-2 border-gray-softest">
+                <div class="border-b-1 border-gray-softest">
```
Comment (rastislav-chynoransky) [inline resources/js/components/About.vue:5]: Check with Filip. Can be either 1 or 2px wide.

**Review by rastislav-chynoransky — CHANGES_REQUESTED** [review]

File: resources/js/components/ConfirmButton.vue (hunk 2 above)
Comment (igor-kamil) [reply resources/js/components/ConfirmButton.vue:3]: yes. I proof-checked that. But in any case - @filipruisl can confirm it..

File: resources/js/components/About.vue (hunk 6 above)
Comment (igor-kamil) [reply resources/js/components/About.vue:5]: it's 1px in design. but yes. this should be proof-checked with @filipruisl

Comment (rastislav-chynoransky) [reply resources/js/components/About.vue:5]: Screenshot 2022-12-09 at 13 47 53
Here it's doubled, however not sure if that was intended...

File: resources/js/views/ItemDetail.vue (hunk 4 above)
Comment (igor-kamil) [reply resources/js/views/ItemDetail.vue:64]: it is `10px` = `mr-2.5`
look:

Snímka obrazovky 2022-12-09 o 14 10 36

**Review by rastislav-chynoransky — APPROVED** [review]

**Pull request merged**